rts/Lua/LuaTextures.cpp

```diff
@@ -13,33 +14,35 @@
 #include "fmt/format.h"
 
 
-const spring::unordered_map<GLenum, GLenum> LuaTextures::Format2Query =
-{
-	{ GL_TEXTURE_1D                  , GL_TEXTURE_BINDING_1D				   },
-	{ GL_TEXTURE_2D                  , GL_TEXTURE_BINDING_2D				   },
-	{ GL_TEXTURE_3D                  , GL_TEXTURE_BINDING_3D				   },
-//	{ GL_TEXTURE_1D_ARRAY            , GL_TEXTURE_BINDING_1D_ARRAY			   },
-	{ GL_TEXTURE_2D_ARRAY            , GL_TEXTURE_BINDING_2D_ARRAY			   },
-//	{ GL_TEXTURE_RECTANGLE           , GL_TEXTURE_BINDING_RECTANGLE			   },
-	{ GL_TEXTURE_CUBE_MAP            , GL_TEXTURE_BINDING_CUBE_MAP			   },
-//	{ GL_TEXTURE_BUFFER              , GL_TEXTURE_BINDING_BUFFER			   },
-	{ GL_TEXTURE_2D_MULTISAMPLE      , GL_TEXTURE_BINDING_2D_MULTISAMPLE	   },
-//	{ GL_TEXTURE_2D_MULTISAMPLE_ARRAY, GL_TEXTURE_BINDING_2D_MULTISAMPLE_ARRAY },
-};
+namespace Impl {
+	static inline bool IsValidLuaTextureTarget(GLenum target) {
```

Comment:
Please convert it to whitelist, rather than blacklist implementation as it's now.

Reply:
Ok!